### DISCUSSION TITLE
Bug in table tool?

### DISCUSSION THREAD

**Discussion by aimeediehl** [knowledgefutures/pubpub#3076] [Support]
Hi PubPub,

The table tool is behaving erratically in my draft pub (in a Mac google chrome browser). Example 1 shows the header repeating multiple times after clicking the toolbar icon once, then after pasting in text. If I click in the table to try to edit, it pastes in about a dozen more headers per click. The pasted text is also incomplete and there are blank columns inserted.
![table bug 1](https://github.com/pubpub/pubpub/assets/104081219/b0c741b8-03eb-4bf0-93ab-1889b6feed9a)

Example 2 shows a table that won't allow me to merge or split cells. It worked once, then quit.
![Table bug 2](https://github.com/pubpub/pubpub/assets/104081219/29e2f612-5efd-4689-ae5b-c82f68b68831)

The pasted text is from an InDesign file; I've never had this issue before.

I've emailed help with this information as well since I'm pretty desperate!

Thank you!

**Comment by eastofesten** [comment]
Adding video for support re: table icon not appearing for merging & splitting cells.


https://github.com/pubpub/pubpub/assets/8959416/420348a4-c62f-4f14-8ee2-5bd31c9a2f8b

**Reply by aimeediehl** [reply]
Thanks for your help with this. I've got things in good enough shape now to go forward if as-is if needed, but ideally I'd still love to merge a few cells so there aren't blank spaces (see Table 4 in the pub).

**Comment by aimeediehl** [comment]
Hi there, I tried inserting a table again today (05/01/2024) in an [old test pub](https://jeed.pubpub.org/pub/5t3rv0zf/draft) and the cell merge and split functions still aren't working. I get the same behaviors in Crome and Safari browsers. I'm on a MacBook Pro M2 Max running Sonoma 14.4.1.

Any updates?

**Reply by eastofesten** [reply]
Hi Aimee - rewatching your screenshare, I'm noticing that the table button from the toolbar (upper right) is no longer colored when you highlight multiple cells. Are you able to click that button while you have multiple cells highlighted?

**Comment by aimeediehl** [comment]
When I highlight multiple empty cells, I can click the table button in the toolbar and it turns green, but nothing else happens. It doesn’t present a dropdown, make the popup appear, or give me any options for manipulating the cells.

The same happens when I select a single empty cell.

__

Aimee Diehl (she/her)
Editorial Director – Journals Division, UVM Press
Production Editor, Journal of Ecological Engineering Design

University of Vermont Libraries
University of Vermont
Howe Library | 538 Main St.
Burlington, VT 05405

(802) 656-3294 | ***@***.******@***.***>
www.uvm.edu/press<https://www.uvm.edu/press> | jeed.pubpub.org<http://jeed.pubpub.org/>
Schedule a Teams meeting with ***@***.***?anonymous&ep=pcard>

[University of Vermont logo: A white letter V outlined by a dark green shield and the text University of Vermont]<https://uvm.edu/>

UVM is subject to the Vermont Public Records Act and communications to and from this email address, including attachments, are subject to disclosure unless exempted under the Act or otherwise applicable law.



From: Emily Esten ***@***.***>
Date: Thursday, May 2, 2024 at 9:46 AM
To: pubpub/pubpub ***@***.***>
Cc: Aimee Diehl (she/her) ***@***.***>, Author ***@***.***>
Subject: Re: [pubpub/pubpub] Bug in table tool? (Discussion #3076)

Hi Aimee - rewatching your screenshare, I'm noticing that the table button from the toolbar (upper right) is no longer colored when you highlight multiple cells. Are you able to click that button while you have multiple cells highlighted?

**Reply by eastofesten** [reply]
When you have a moment, can you add a screenshare of that experience?

**Reply by aimeediehl** [reply]
Here's a screenshare. It starts out not working, works without explanation at :39, then stops working again in a different row around 1:29. LMK if the compressed file causes problems and I'll email it instead.
[Table bug 050224 480p.mov.zip](https://github.com/pubpub/pubpub/files/15189846/Table.bug.050224.480p.mov.zip)
